lldb/tools/lldb-dap/LLDBUtils.h

```diff
@@ -159,6 +159,9 @@ GetEnvironmentFromArguments(const llvm::json::Object &arguments);
 /// Take ownership of the stored error.
 llvm::Error ToError(const lldb::SBError &error);
 
+/// Provides the string value if this data structure is a string type.
+std::string GetStringValue(const lldb::SBStructuredData &data);
```

Comment:
There's a few more places that can use this helper. I'm planning on addressing that in a follow-up PR.